sql/storm-sql-core/pom.xml

```diff
@@ -66,10 +66,34 @@
             <artifactId>calcite-core</artifactId>
```

Comment:
I'll see how the number is changed. It might be remain the same if I fixed checkstyle issue from A and violated from B. 
Actually I'm focusing on planning and addressing window aggregation and join, so TBH I'd like to address the checkstyle issue later altogether.

Reply:
Sure. I didn't mean to say that you should do any extra work on checkstyle in this, just wanted to make sure reducing the counter wasn't being forgotten by accident.

Reply:
Good. Thanks for the reminder. :)